### PR TITLE
move DLP-based Schnorr signature to Module 2

### PR DESCRIPTION
**Pull request by jae-cuz**
### Description

Distinguishing Schnorr signatures based on DLP and EC in Module 3, I suggest to move DLP-based Schnorr signature to Digital Signature section in Module 2.

### Related Issues

#4 

### Affected Tutorials

- Module Name: Module 2 & Module 3
- Section/Step: Digital Signatures / Schnorr Signatures and EdDSA

### Type of Change

- [ ] Typo/grammar fix
- [ ] Code correction (e.g., a bug in a code snippet)
- [ ] New tutorial or section
- [x] Other (please describe): section rearrangement

### Testing

I read the changed part of the document again.

### Checklist

- [x] I have performed a self-review of my own changes.
- [x] I have made corresponding changes to related documentation.

### CODE REVIEW

File: MODULE_3.md [hunk 1]
```diff
@@ -21,15 +21,13 @@ The following StackExchange answer goes over the differences between some of the
 
 EdDSA modifies and extends the Schnorr signature scheme to provide additional benefits. So it is crucial to first get an understanding of Schnorr signatures.
 
-### Schnorr Signatures
+### EC-Schnorr Signatures
 
-Schnorr signatures are a digital signature scheme known for their simplicity and efficiency. A key benefit is their linear structure, which allows for multi-signature aggregation.
+Schnorr signatures also can be extended to elliptic curve. A key benefit is their linear structure, which allows for multi-signature aggregation.
```
Comment (adrianmcli) [inline MODULE_3.md:26]:
```suggestion
Schnorr signatures can also be extended to elliptic curves. A key benefit is their linear structure, which allows for multi-signature aggregation.
```

**Review by adrianmcli — CHANGES_REQUESTED** [review]
Just a slight grammatical change. Thanks!

**Comment by jae-cuz** [comment]
Sure :) Thanks for catching that!

**Comment by adrianmcli** [comment]
thanks @jae-cuz
btw, if you write "Fixes #4" in the PR description, the merging of the PR will automatically close the issue as well.

See here for more info: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Pull request merged**